packages/rancher-istio/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: local
-packageVersion: 01
-releaseCandidateVersion: 02
+packageVersion: 00
+releaseCandidateVersion: 01
```

Comment:
I think this has to be `00`
CC: @aiyengar2

Reply:
I had an error on my initial pr, so bumped the rc version. I ave now put it back to 00. You are correct, it should be 00.